samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -523,31 +530,38 @@ public virtual Response StartTranslation(RequestContent content, RequestOptions
             {
                 message.SetProperty("RequestOptionsPerCallPolicyCallback", options.PerCallPolicy);
             }
-            using var scope = _clientDiagnostics.CreateScope("DocumentTranslationClient.StartTranslation");
-            scope.Start();
-            try
+
+            Response BeginOperation()
```

Comment:
Why do we need a local function?

Reply:
We don't need this to be a local function, we could either "inline" the body or generate this as a separate method on the client (but mark it as private).  I preferred the local function stylistically, I think because I knew this was the only place that would ever want to call it.  And unlike the "DataPlane" version of this code, we don't have a separate "RestClient" where we can stick this methods.

FWIW, I was playing a similar trick with paging, where I would have local functions for both the initial request and the "get next page" request.  If you'd prefer these just all be private functions on the main class, happy to make that change.

Reply:
The reason I asked is that it seemed to me that we should be able to just inline the code.

Reply:
> The reason I asked is that it seemed to me that we should be able to just inline the code.

Yeah - makes sense and I can do it easily here - will fix.